Server/config/helpers.js

```diff
@@ -1,5 +1,5 @@
 var fs = require('fs');
-var gm = require('gm');
+var gm = require('gm').subClass({imageMagick: true});
```

Comment:
not including `.subClass({imageMagick: true});` was giving me those weird spawn errors. this fixed it.